packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -4,12 +4,19 @@ import {
   BUILD_MANIFEST,
   ROUTE_NAME_REGEX,
   IS_BUNDLED_PAGE_REGEX,
+  CLIENT_STATIC_FILES_PATH,
   CLIENT_STATIC_FILES_RUNTIME_MAIN,
 } from 'next-server/constants'
 
 // This plugin creates a build-manifest.json for all assets that are being output
 // It has a mapping of "entry" filename to real filename. Because the real filename can be hashed in production
 export default class BuildManifestPlugin {
+  options: { buildId: string; clientManifest: boolean }
```

Comment:
I'd prefer to see this as separate fields.
```suggestion
  private buildId: string
  private clientManifest: boolean
```